### PR TITLE
feat(ui): Add error dialog component

### PR DESCRIPTION
**Pull request by bdauria**

### CODE REVIEW

File: src/MaterialComponents/Dialog/Error/index.test.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      <ErrorDialog
+        open
+        title="Error"
+        text="Something unexpected happened..."
+        confirmLabel="Close"
+        onClose={() => {}}
+      />,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
```
Comment (kduret) [inline src/MaterialComponents/Dialog/Error/index.test.jsx]: Avoid to get an empty snapshot :
```suggestion
    expect(container).toMatchSnapshot();
```

File: package.json [hunk 2]
```diff
@@ -89,7 +89,6 @@
     "eslint-config-airbnb": "^17.1.0",
     "eslint-config-prettier": "^4.3.0",
     "eslint-plugin-import": "^2.17.3",
-    "eslint-plugin-jest": "^22.7.2",
```
Comment (kduret) [inline package.json]: why do you remove this dependency ?

File: src/MaterialComponents/Dialog/Error/index.test.jsx (hunk 1 above)
Comment (bdauria) [reply src/MaterialComponents/Dialog/Error/index.test.jsx]: > Avoid to get an empty snapshot :

It doesn't really help, we just get an empty container instead. I think the problem comes from the mui animation, I'll leave a get around this later on.

File: package.json (hunk 2 above)
Comment (bdauria) [reply package.json]: I made a fix in eslint-config-centreon. There is a property that can be set in the env object to allow just namespace. see https://github.com/centreon/eslint-config-centreon/commit/d196ef77510f220affed3b82e12d160a9974d87a

**Pull request merged**